app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -1230,6 +1238,11 @@ private void destroyAdapter(MoPubRecyclerAdapter adapter) {
     }
   }
 
+  @Override public void startCatappultWebView() {
+    CustomTabsHelper.getInstance()
+        .openInChromeCustomTab("https://catappult.io/", getContext(), theme);
+  }
+
```

Comment:
This could be extracted somewhere to center the logic related to catappult navigation. At the moment we have this method copy pasted here and in appcoinsinfofragment